pysat/instruments/sw_f107.py

```diff
@@ -383,8 +382,14 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
             if data.empty:
                 warnings.warn("no data for {:}".format(date), UserWarning)
             else:
-                times = [pysat.datetime.strptime(time, '%Y %m %d')
-                         for time in data.pop('time')]
+                try:
+                    # This is the new data format
+                    times = [pysat.datetime.strptime(time, '%Y%m%d')
+                             for time in data.pop('time')]
+                except ValueError:
```

Comment:
Is the old format still in use on the website?  Poking around with (quasi-)random dates, it looks like the new version is the only one. Tried Jan 1 1950, 1970, 2018.  Lines 389-392 uncovered in coveralls.

Reply:
I don't think it is, but that doesn't mean that people won't have the old files poking around on their computer.  I thought it'd be safer to allow both.  Happy to change it.

Reply:
Since this is just for the download routine, I don't think this is needed.

Reply:
Removed this, as suggested.